client/src/modules/list/index.jsx

```diff
@@ -110,35 +141,23 @@ class List extends Component {
       return null;
     }
 
-    const {
-      cohortId,
-      isArchived,
-      isGuest,
-      isMember,
-      isOwner,
-      name,
-      type
-    } = list;
+    const { cohortId, isArchived, isMember, isOwner, name, type } = list;
     const isCohortList = cohortId !== null;
 
     return (
       <Fragment>
-        <Toolbar>
-          {cohortId && !isGuest && (
-            <ToolbarLink
-              additionalIconSrc={ArrowLeftIcon}
-              mainIcon={<CohortIcon />}
-              path={`/cohort/${cohortId}`}
-              title="Go back to cohort"
-            />
-          )}
-        </Toolbar>
+        <Toolbar />
```

Comment:
I think that the toolbar should be moved to the Layout component now. All components render the toolbar but don't use it in any way.